samples/will_it_shuffle.lobster

```diff
@@ -0,0 +1,228 @@
+// adaptation of https://bost.ocks.org/mike/shuffle/compare.html
+//
```

Comment:
You can add yourself as the author, seeing as its one of the few files not by me.